CHANGELOG.md

```diff
@@ -2,6 +2,41 @@ Changes by Version
 ==================
 <!-- next version -->
 
+## 0.93.0
+
+### 🛑 Breaking changes 🛑
+
+  This version brings stable semantic convention for network and HTTP. It includes new metrics and attributes names.
+- `target allocator`: Use standard K8s label selectors for Prometheus CRs in target allocator config (#1907)
+  This is a breaking change only for users of standalone target allocator. Operator users are unaffected.
+  The operator is still compatible with previous target allocator versions, and will be for the next 3 releases.
+- `target allocator`: Use recommended interfaces(resource selector) by the prometheus-operator for watching CRs. (#2309)
+  The target allocator now requires get/list/watch permissions for namespaces. Update your RBAC permissions for the attached role, if necessary.
+
+### 💡 Enhancements 💡
+
+- `operator`: Add SharedProcessNamespace capabilities to the OpenTelemetryCollector CRD (#2472)
+- `opamp bridge`: Added env variable parsing to opamp bridge config loading (#2577)
+- `target allocator`: Set the default filter strategy in the CRD (#2477)
+- `operator, target allocator`: Adding a feature flag to disable default prometheus annotations (#2554)
+- `operator`: Switch ()
```

Comment:
Looks like something broke here.

Reply:
mh... Seems its not coming form any of the changelogs we had in `.chloggen`.